woodwork/table_accessor.py

```diff
@@ -67,7 +67,9 @@ def init(self,
                 specified logical type for the column. Defaults to True.
             column_descriptions (dict[str -> str], optional): Dictionary mapping column names to column descriptions.
             schema (Woodwork.Schema, optional): Typing information to use for the DataFrame instead of performing inference.
-                Any other arguments provided will be ignored.
+                Any other arguments provided will be ignored. Note that any changes made to the typing information
+                after passing in the typing information will propogate to the original object. It is not advised to
```

Comment:
Hmm. Is this true now assuming the original schema was accessed through `df.ww.schema`? Maybe I'm getting tripped up by the work `original` here. If I'm understanding things correctly these hold true:
- If a user initializes with a schema object obtained from another dataframe, and then makes changes to the schema after initialization, the changes to the schema will propagate to the second object, but not the first object from which the schema originated
- If any two or more dataframes are initialized with the same schema object, changes to one will be propagated to the others

If I'm following properly, maybe we should update this to something like: 
```
Any other arguments provided will be ignored. Note that any changes made to the schema object after
initialization will propagate to the DataFrame. Similarly, to avoid unintended typing information changes,
the same schema object should not be shared between DataFrames.
```

Reply:
Yes both of those are correct, and I think your suggestion displays that really well. I'll change.